packages/react-select/src/__tests__/Select.test.js

```diff
@@ -2765,3 +2765,17 @@ test('renders with custom theme', () => {
     window.getComputedStyle(firstOption).getPropertyValue('background-color')
   ).toEqual(primary);
 });
+
+test('filterOption only called once per option when opening menu', () => {
```

Comment:
I'm not sure how I would go about testing the memoization of buildCategorizedOptions. So I test how many times filterOption gets called when opening menu. Don't know if this is necessary though.